packages/luna/src/components/Button/Button.tsx

```diff
@@ -44,7 +44,7 @@ export const Button: React.FC<{
         "tw-relative tw-font-cairo tw-cursor-pointer tw-font-medium",
         "tw-text-center tw-font-normal tw-transition-colors tw-ease-out tw-duration-200 tw-rounded-lg",
         "tw-outline-none tw-transition-all tw-outline-0",
-        "tw-w-fit tw-flex tw-items-center tw-justify-center tw-text-base",
+        "tw-flex tw-items-center tw-justify-center tw-text-base",
```

Comment:
The button is designed to fit the width of its content. If you want it to take the full width you can extend it by `tw-w-full` class.